### PR TITLE
Update the naming of openshift on rhv to ovirt

### PR DESCRIPTION
**Pull request by sradco**
Updated all references of rhv to use ovirt insread,
since this is an upstream repo.

Signed-off-by: Shirly Radco <sradco@redhat.com>

Please review: @yanivlavi @cwilkers

### CODE REVIEW

File: roles/openshift_ovirt/defaults/main.yml [hunk 1]
```diff
@@ -1,51 +1,51 @@
 ---
-# Defaults for oVirt/RHV provider integration
+# Defaults for oVirt/oVirt provider integration
```
Comment (cwilkers) [inline roles/openshift_ovirt/defaults/main.yml]: Here's a case where the sed did not make sense. It may need to be reduced to just oVirt

Comment (sradco) [reply roles/openshift_ovirt/defaults/main.yml]: Thanks, I updated

**Pull request closed without merging**